src/Features/CSharp/Portable/ChangeAccessibilityModifier/CSharpChangeAccessibilityModifierCodeFixProvider.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Composition;
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.ChangeAccessibilityModifier;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.CSharp.ChangeAccessibilityModifier
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.ChangeAccessibilityModifier), Shared]
+    internal class CSharpChangeAccessibilityModifierCodeFixProvider : AbstractChangeAccessibilityModifierCodeFixProvider
```

Comment:
For ordering I think you could apply ExtensionOrder attribute here

Reply:
Yes I mentioned it, but I'm not sure which order should I choose.